more_itertools/more.py

```diff
@@ -175,6 +197,55 @@ def peek(self, default=_marker):
                 return default
         return self._cache[0]
 
+    def prepend(self, *items):
+        """Stack up items to be the next ones returned from ``next()`` or
+        ``self.peek()``. The items will be returned in first-in first-out order:
+
+            >>> p = peekable([1, 2, 3])
+            >>> p.prepend(10, 11, 12)
+            >>> next(p)
+            10
+            >>> list(p)
+            [11, 12, 1, 2, 3]
+
+        If you want to prepend elements such that the last one in the argument
```

Comment:
Could you remove this block? Starting with "If you want to prepend elements..." and ending with "...could be wrapped in a `for` loop". I don't think we need to remind people that to reverse something that they should use `reverse`... :business_suit_levitating: 

Also, the example just above makes it clear how things work.